examples/DotNetCore/WebDemoNet6/WebDemoNet6/Pages/Beta.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿@page
+@model BetaModel
+@{
+    ViewData["Title"] = "Beta Feature";
+}
+<h1>@ViewData["Title"]</h1>
+
+<p>The beta feature goes here.</p>
```

Comment:
It looks like this page is only loaded if beta is enabled. So shouldn't the message be "Beta is enabled?"

Reply:
Updated the text to match existing feature management quickstart documentation. Thanks!